### DISCUSSION TITLE
Use previous simulation result for next time step

### DISCUSSION THREAD

**Discussion by hokonkim** [idaholab/moose#20169] [Q&A General] [answered]
Hello, I would like to know a way to use previous simulation result for the next time step.

Here is the situation that I have

Transient: 0 -> 0.1 -> 0.2 -> 0.4, can be done in just one simulation
Steady: 0 -> 0.1 /  0.1 -> 0.2 / 0.2 -> 0.4, can bedone in "Three" simulations

I want to know how to inherit previous simulation status for the new(next) simulation.

In other words, how to initialize variables and conditions by using previous simulation result?

**Comment by ykvishal** [comment, marked as answer]
Have a look at [SolutionUserObject](https://mooseframework.inl.gov/source/userobject/SolutionUserObject.html). Also, see [Restarting and recovering from previous simulations](https://mooseframework.inl.gov/modules/porous_flow/restart.html) page.  There will be some posts on this topic.
